src/main/java/com/minecolonies/coremod/commands/colonycommands/CommandRaidNow.java

```diff
@@ -48,7 +79,18 @@ public String getName()
     @Override
     public LiteralArgumentBuilder<CommandSource> build()
     {
+        String[] s = new String[5];
```

Comment:
This should be probably handled from some kind of registry or at least a list of raid types

Reply:
If a registry, any preferences on where to put it?  It's not exactly com.minecolonies.api.entities or com.minecolonies.api.colony.colonyEvents, but feels a little ostentatious making a separate package like com.minecolonies.api.colony.raidTypes to carry five strings.